### DISCUSSION TITLE
Can we make element of block modifier?

### DISCUSSION THREAD

**Discussion by learningcreature** [getbem/getbem.github.io#250] [Q&A]
Hi there, I just wanted to know whether is following code is valid or not? Creating an element for the block modifier i.e. in this case "block--mod" is a modifier for the "block" block. Is it allowed to create a nested element with this pattern: "block--mod__elm". I am quite new to BEM and trying to understand it. 

```
<div class="block block--mod">
   <div class="block__elm block--mod__elm">
</div>
```

**Comment by garrypolley** [comment]
I think you're wanting the `block__elm--mod`

```html
<div class="block block--mod">
   <div class="block__elm block__elm--mod">
</div>
```

My understanding is that the _only_ order is Block -> Element -> Modifier.  The element and/or modifier can be left off, but an element cannot ever follow a modifier. 

From a CSS standpoint I have written mod classes that were wildcarded. Such as `--full-width`

**Comment by himedlooff** [comment]
More chatter here https://github.com/getbem/getbem.com/issues/184
